data/json/itemgroups/Locations_MapExtras/locations_commercial.json

```diff
@@ -895,6 +895,8 @@
       { "item": "folded_bicycle", "prob": 3 },
       { "item": "sunglasses", "prob": 25 },
       { "item": "cowboy_hat", "prob": 25 },
+      { "item": "mp3", "prob": 35, "charges": [ 0, 100 ] },
```

Comment:
Is this intended change?

Reply:
May have been unintentional, as I said previously I don't remember tinkering with the mp3 item while making these variants a couple of months ago. But it doesn't t look like anything is wrong here unless I'm missing something. If I am, please don't be afraid to tell me and I'll do my best to fix it

Reply:
There is already mp3 exist in this itemgroup, just two lines below. So this will be double, do we need it?

Reply:
oh yeah nah we dont need a double of it. sorry didn't catch that earlier